rpc/flipt/validation.go

```diff
@@ -369,6 +369,33 @@ func (req *DeleteSegmentRequest) Validate() error {
 	return nil
 }
 
+const MAX_JSON_ARRAY_ITEMS = 100
```

Comment:
I've hard coded it to 100 items. Let me know if this can be handled better or if we want a different limit.

Reply:
100 sounds good to me!

Reply:
Yeah 100 sounds good to me as well. Might be able to make this configurable in the future.